src/AppInstallerCLICore/Workflows/ConfigurationFlow.cpp

```diff
@@ -1246,17 +1243,58 @@ namespace AppInstaller::CLI::Workflow
             return unit;
         }
 
-        ConfigurationUnit CreatePowerShellModuleGetUnit(const std::wstring& moduleName)
+        ConfigurationUnit CreatePowerShellPackageUnit()
         {
-            ConfigurationUnit unit = CreateConfigurationUnitFromUnitType(s_UnitType_PowerShellModuleGet, Utility::ConvertToUTF8(moduleName));
+            ConfigurationUnit unit = CreateConfigurationUnitFromUnitType(s_UnitType_WinGetPackage_DSCv3, "Microsoft.PowerShell");
 
             ValueSet settings;
-            settings.Insert(s_Setting_PowerShellGet_ModuleName, PropertyValue::CreateString(moduleName));
+            settings.Insert(s_Setting_WinGetPackage_Id, PropertyValue::CreateString(L"Microsoft.PowerShell"));

```

Comment:
Constants?

Also, we should consider extracting this from / filtering out of the package results.